proposals/dao/peg_stability_module.ts

```diff
@@ -0,0 +1,233 @@
+
+// Constants for deploy. Tweak as needed.
+const daiPSMMintFeeBasisPoints = 30;
+const daiPSMRedeemFeeBasisPoints = 30;
+
+const wethPSMMintFeeBasisPoints = 100;
+const wethPSMRedeemFeeBasisPoints = 100;
+
+const daiReservesThreshold = ethers.utils.parseEther('30000000');
+const wethReservesThreshold = ethers.utils.parseEther('7500');
+
+const daiFeiMintLimitPerSecond = ethers.utils.parseEther('10000');
+const wethFeiMintLimitPerSecond = ethers.utils.parseEther('10000');
+
+const daiPSMBufferCap = ethers.utils.parseEther('10000000');
+const wethPSMBufferCap = ethers.utils.parseEther('10000000');
+
+const daiDecimalsNormalizer = 0;
+const wethDecimalsNormalizer = 0;
+
+const daiFloorPrice = 9_500;
+const daiCeilingPrice = 10_500;
+
+// PCVDrip Controller Params
+
+// drips can happen every hour
+const dripFrequency = 3_600;
+
+// do not incentivize these calls
+const incentiveAmount = 0;
+
+const daiDripAmount = ethers.utils.parseEther('5000000');
+const wethDripAmount = ethers.utils.parseEther('1250');
+
+const toBN = ethers.BigNumber.from;
+
+// Do any deployments
+// This should exclusively include new contract deployments
+const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: NamedAddresses, logging: boolean) => {
+  const {
+    core,
+    fei,
+    dai,
+    weth,
+    chainlinkDaiUsdOracleWrapper,
+    chainlinkEthUsdOracleWrapper,
+    compoundDaiPCVDeposit,
+    aaveEthPCVDeposit
+  } = addresses;
+
+  if (!core) {
+    throw new Error('Core address not set.');
+  }
+
+  const daiPSMFactory = await ethers.getContractFactory('PriceBoundPSM');
+  const wethPSMFactory = await ethers.getContractFactory('PegStabilityModule');
+  const psmRouterFactory = await ethers.getContractFactory('PSMRouter');
+  const PCVDripControllerFactory = await ethers.getContractFactory('PCVDripController');
+
+  // Deploy DAI Peg Stability Module
+  // PSM will trade DAI between 98 cents and 1.02 cents.
+  // If price is outside of this band, the PSM will not allow trades
+  const daiPSM = await daiPSMFactory.deploy(
+    daiFloorPrice,
+    daiCeilingPrice,
+    {
+      coreAddress: core,
+      oracleAddress: chainlinkDaiUsdOracleWrapper,
+      backupOracle: chainlinkDaiUsdOracleWrapper,
+      decimalsNormalizer: daiDecimalsNormalizer,
+      doInvert: false
+    },
+    daiPSMMintFeeBasisPoints,
+    daiPSMRedeemFeeBasisPoints,
+    daiReservesThreshold,
+    daiFeiMintLimitPerSecond,
+    daiPSMBufferCap,
+    dai,
+    compoundDaiPCVDeposit
+  );
+
+  // Deploy ETH Peg Stability Module
+  const wethPSM = await wethPSMFactory.deploy(
+    {
+      coreAddress: core,
+      oracleAddress: chainlinkEthUsdOracleWrapper,
+      backupOracle: chainlinkEthUsdOracleWrapper,
+      decimalsNormalizer: wethDecimalsNormalizer,
+      doInvert: false
+    },
+    wethPSMMintFeeBasisPoints,
+    wethPSMRedeemFeeBasisPoints,
+    wethReservesThreshold,
+    wethFeiMintLimitPerSecond,
+    wethPSMBufferCap,
+    weth,
+    aaveEthPCVDeposit
```

Comment:
do we want this to be a splitter?

Reply:
Why would we want it to be a splitter? It makes sense to have the pcv deposit and pcv dripper pulling and pushing to the same place.